packages/web/src/components/DatePicker/DatePicker.tsx

```diff
@@ -140,8 +140,9 @@ export const DatePicker: React.FC<Props> = ({
                 </ChangeDayButtonsStyledFlex>
 
                 {withTodayButton &&
-                  formattedCurrentMonth !== formattedSelectedMonth && (
+                //removed the condition here and put it in the prop 'isCurrentDate to check for truthyness
                     <TodayStyledText
+                    isCurrentDate={formattedCurrentMonth === formattedSelectedMonth}
```

Comment:
nit-picky request: please keep `isCurrentDate` aligned with the other props. Looks like it just needs a few extra spaces to make that happen